dali/kernels/imgproc/warp/jitter.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_IMGPROC_WARP_JITTER_H_
+#define DALI_KERNELS_IMGPROC_WARP_JITTER_H_
+
+#include <random>
+#include "dali/core/geom/vec.h"
+
+namespace dali {
+namespace kernels {
+
+template <int dim>
+struct JitterMappingCPU {
+  ivec<dim> operator()(const ivec<dim> &v) const {
+    return v;
```

Comment:
Could you explain, how is this supposed to work, since it just returns the input? I don't really understand it